spec/rpc/README.md

```diff
@@ -848,6 +849,42 @@ curl -X POST https://localhost:26657 -d "{\"jsonrpc\":\"2.0\",\"id\":1,\"method\
 }
 ```
 
+### GenesisChunked
+
+Get the genesis document in a chunks to support easily transfering larger documents.
+
+#### Parameters
+
+- `chunkID`: the index number of the chunk that you wish to fetch.
```

Comment:
```suggestion
- `chunkID (integer)`: the index number of the chunk that you wish to fetch.
```

Reply:
It should also state if it's one-based or zero-based.